src/Infrastructure/ContainerBuilder.php

```diff
@@ -187,7 +194,13 @@ public function buildContainer(OutputInterface $output, $intSize)
             return new StandardPrettyPrinter();
         };
         $container['resultPrinter'] = function ($c) {
-            return new CLIResultPrinter($c['output'], $c['nodePrinter'], $c['nodeStatementsRemover']);
+            switch ($this->outputFormat) {
```

Comment:
Unfortunately `$this` cannot be used in a closure, because we still have to support PHP 5.3